src/App.jsx

```diff
@@ -14,70 +15,101 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+let counter = 0;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const fullGoods = goods.map((good) => {
+  counter += 1;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return ({
+    oneGood: `${good}`,
+    id: counter,
+  });
+});
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const styleForTable = 'has-background-success-light';
+  const styleForButton = 'button is-info';
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const resetButton = () => () => setSelectedGood('');
+  const setButton = value => () => setSelectedGood(value);
```

Comment:
Not really descriptive names. We have a convention to start our handler function starting from `handle` or `on`, so
`handleResetSelected` and `handleSetSelected`